codelabs/codelab-02/README.md

```diff
@@ -12,11 +12,11 @@ In this codelab, you'll get to play around with S3.
 
 ### Setting Up
 
-Before starting this codelab, run `git pull` in the `389Lfall18` directory to update your local copy of the class repository.
+Before starting this codelab, run `git pull` in the `389Lspring18` directory to update your local copy of the class repository.
 
 ### Getting Started with AWS
 
-The first step is going to be to get an [AWS account](https://aws.amazon.com (https://aws.amazon.com/)). If you already have an Amazon.com (http://amazon.com/) account with your umd.edu (http://umd.edu/) email, then you can go ahead and sign in with the same account. Sign in or Sign up [here](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html?nc2=h_ct).
+The first step is going to be to get an [AWS account](https://aws.amazon.com). If you already have an Amazon.com (http://amazon.com/) account with your umd.edu (http://umd.edu/) email, then you can go ahead and sign in with the same account. Sign in or Sign up [here](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html?nc2=h_ct).
```

Comment:
Use [Amazon.com] (https://amazon.com)  (minus the extra space) to get [Amazon.com](https://amazon.com) etc.